mysql-test/include/search_pattern_in_file.inc

```diff
@@ -82,14 +82,41 @@ perl;
 
     $ENV{SEARCH_FILE} =~ s{^.*?([^/\\]+)$}{$1};
 
+    my $print_func= sub { print($_[0]); };
+    my $die_func= sub { die($_[0]); };
+    my $abort_func;
+    my $match_func;
+
+    # The outcome of matching is dependent on the context that we are called.
+    #
+    # 1) In the normal case (SEARCH_ABORT_IS_SUCCESS is NOT set), when the
```

Comment:
I could not locate any example of SEARCH_ABORT_IS_SUCCESS.. Did I overlook?

At any rate this env variable suits for mentioning in the header of this file. Please consider that.

Reply:
It is used in `wait_for_pattern_in_file.inc` [here](https://github.com/MariaDB/server/pull/3605/commits/f46a34e77b80bd48f7b60a13c2ff9fcd3a9262ef#diff-40d389db838c160e05682e40361fda980a38af1e9f8c46b7e6eb4e38c6d222b6R39)

As far as mentioning in the header file, agreed, will add that.